### ISSUE TITLE
CSSMathNegate() constructor page is a stub

### ISSUE THREAD

**Issue by queengooborg** [mdn/content#28118]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/CSSMathNegate/CSSMathNegate

### What information was incorrect, unhelpful, or incomplete?

The page for the `CSSMathNegate()` constructor is a stub and need to be updated.

### What did you expect to see?

N/A

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/cssmathnegate/cssmathnegate`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/CSSMathNegate/CSSMathNegate
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/cssmathnegate/cssmathnegate/index.md
* Last commit: https://github.com/mdn/content/commit/d76defab4ca13261e9de81ae1df125345f847b0a
* Document last modified: 2023-04-07T05:46:57.000Z

</details>

**Comment by hamishwillee** [comment]
Yes, and many other flaws. Working my way through in https://github.com/mdn/content/issues/44849#issuecomment-5100621001
